examples/arkode/F2003_parallel/ark_brusselator1D_task_local_nls_f2003_--monitor_--explicit_--tf_3.out

```diff
@@ -23,50 +23,50 @@ Integrator Options:
  1.12500E+00     1.69428E+00     2.64957E+00     3.49997E+00
- 1.20000E+00     1.72515E+00     2.57279E+00     3.49996E+00
- 1.27500E+00     1.74317E+00     2.50340E+00     3.49997E+00
- 1.35000E+00     1.74838E+00     2.44230E+00     3.49998E+00
+ 1.20000E+00     1.72515E+00     2.57279E+00     3.49998E+00
+ 1.27500E+00     1.74317E+00     2.50340E+00     3.49998E+00
+ 1.35000E+00     1.74838E+00     2.44230E+00     3.49997E+00
  1.42500E+00     1.74164E+00     2.38997E+00     3.49997E+00
- 1.50000E+00     1.72432E+00     2.34665E+00     3.49996E+00
+ 1.50000E+00     1.72432E+00     2.34665E+00     3.49997E+00
  1.57500E+00     1.69793E+00     2.31238E+00     3.49997E+00
- 1.65000E+00     1.66399E+00     2.28715E+00     3.49998E+00
+ 1.65000E+00     1.66399E+00     2.28715E+00     3.49996E+00
  1.72500E+00     1.62387E+00     2.27087E+00     3.49997E+00
  1.80000E+00     1.57879E+00     2.26339E+00     3.49997E+00
- 1.87500E+00     1.52981E+00     2.26452E+00     3.49998E+00
- 1.95000E+00     1.47785E+00     2.27400E+00     3.49998E+00
- 2.02500E+00     1.42369E+00     2.29148E+00     3.49997E+00
+ 1.87500E+00     1.52981E+00     2.26452E+00     3.49997E+00
+ 1.95000E+00     1.47785E+00     2.27400E+00     3.49997E+00
+ 2.02500E+00     1.42369E+00     2.29148E+00     3.49998E+00
  2.10000E+00     1.36803E+00     2.31652E+00     3.49998E+00
- 2.17500E+00     1.31147E+00     2.34859E+00     3.49998E+00
- 2.25000E+00     1.25455E+00     2.38711E+00     3.49997E+00
- 2.32500E+00     1.19774E+00     2.43138E+00     3.49997E+00
- 2.40000E+00     1.14147E+00     2.48069E+00     3.49998E+00
+ 2.17500E+00     1.31147E+00     2.34859E+00     3.49999E+00
+ 2.25000E+00     1.25455E+00     2.38711E+00     3.49999E+00
+ 2.32500E+00     1.19774E+00     2.43138E+00     3.49998E+00
+ 2.40000E+00     1.14147E+00     2.48069E+00     3.49999E+00
  2.47500E+00     1.08607E+00     2.53429E+00     3.49999E+00
- 2.55000E+00     1.03187E+00     2.59142E+00     3.49998E+00
+ 2.55000E+00     1.03187E+00     2.59142E+00     3.49997E+00
  2.62500E+00     9.79114E-01     2.65138E+00     3.49998E+00
- 2.70000E+00     9.27998E-01     2.71349E+00     3.49999E+00
- 2.77500E+00     8.78688E-01     2.77716E+00     3.49998E+00
- 2.85000E+00     8.31306E-01     2.84186E+00     3.49998E+00
+ 2.70000E+00     9.27998E-01     2.71349E+00     3.49998E+00
+ 2.77500E+00     8.78688E-01     2.77716E+00     3.49997E+00
+ 2.85000E+00     8.31306E-01     2.84186E+00     3.49999E+00
  2.92500E+00     7.85951E-01     2.90716E+00     3.50000E+00
  3.00000E+00     7.42699E-01     2.97270E+00     3.49999E+00
  -----------------------------------------------------------
  
 Final Solver Statistics (for processor 0):
-  Steps            = 238744
-  Step attempts    = 239406
-  Error test fails = 662
-  RHS evals        = 718221
+  Steps            = 236112
+  Step attempts    = 332018
+  Error test fails = 95906
+  RHS evals        = 996057
```

Comment:
Steven confirmed this increase in the number of error test failures is fixed by the updated controller defaults in PR #565

Reply:
Fixed by commit 40dd2bb